config-example.properties

```diff
@@ -42,6 +42,9 @@ graph.dataaccess=RAM_STORE
 # graph but less for every request.
 # prepare.chWeighting=fastest
 #
+# You can also setup multiple weightings, by providing a coma separated list like:
+# prepare.chWeighting=fastest, shortest
```

Comment:
I actually don't think that adding logic to getters/setter is a bad practice. As long as we are trying to improve the quality of the "seted" value.

The pro of doing it in the setter is that we have a central/unique place to execute the correction of weighting values.